herbert_core/graphics/@IX_dataset_3d/sliceomatic.m

```diff
@@ -81,6 +81,9 @@
 
 [xlabel,ylabel,zlabel]=make_label(w);
 clim = [min(w.signal(:)) max(w.signal(:))];
+if clim(2) == clim(1)
```

Comment:
I can't see any tests for this failure mode.

Reply:
it fails in GUI and I do not know how to test it.

Reply:
As this is limits for plotting, would it be better to set the limits as `clim(1) - 0.5` and `clim(1) + 0.5` in the case `clim(1) == clim(2)`? That would place the single data value in the centre of the range rather than at one edge

(same for the sliceomaticfigure) fix

Reply:
yes, probably. I will make it consistent whith 2D plottings where +-1  is used in the similar situation.